### ISSUE TITLE
Check for feature importance

### ISSUE THREAD

**Issue by c-schicho** [NoahPichl/MLPC_The_haunted_turtles#4]
- [x] check which features might be of interest for classification and are correlated to the labels

**Comment by c-schicho** [comment]
*idea: check feature importance with a random forest search and plot them with a parallel coordinate plot*

```py
X_train, _, y_train, _ = train_test_split(anot.iloc[:, :-1], anot.iloc[:, -1:], test_size=0.01)

sc = StandardScaler()
sc.fit(X_train)
X_test = pd.DataFrame(sc.transform(X_train), columns=anot.columns[:-1])
X_test = pd.DataFrame(sc.transform(X_test), columns=anot.columns[:-1])

rf_cls = RandomForestClassifier()

rf_cls.fit(X_train, y_train.values.ravel())

sort_indices = np.argsort(rf_cls.feature_importances_)[::-1]
feature_importance = rf_cls.feature_importances_[sort_indices]

n_features = len(anot.columns[:-1])

fig, _ = plt.subplots(figsize=(10, 8))
plt.title('Feature Importance')
plt.bar(range(n_features), feature_importance, align='center')
plt.xticks(range(n_features), X_train.columns[sort_indices], rotation=90)
plt.tight_layout()
plt.show()
```
